### PR TITLE
Add csv data module to the distribution

### PR DESCRIPTION
**Pull request by SasinduDilshara**
Add csv data module to the distribution

https://github.com/ballerina-platform/module-ballerina-data.csv/pull/4 will contains the more details about CSV data module.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -55,6 +55,7 @@ stdlibMqttVersion=1.1.1
 stdlibOAuth2Version=2.11.0
 stdlibTomlVersion=0.5.1
 stdlibYamlVersion=0.5.3
+stdlibDataCsvVersion=0.1.0
```
Comment (ThisaruGuruge) [inline gradle.properties:58]: Shall we use a timestamped version here and check whether the build is passing?

**Comment by ThisaruGuruge** [comment]
Aren't we releasing this with the Update 10 distribution?

**Comment by SasinduDilshara** [comment]
> Aren't we releasing this with the Update 10 distribution?

We are planned to release 0.1.0 into the Ballerina central and release this with U11.

File: gradle.properties (hunk 1 above)
Comment (SasinduDilshara) [reply gradle.properties]: Added in https://github.com/ballerina-platform/ballerina-distribution/pull/5528/commits/1546f3612e4d807c6e7ac027bde3e1b180aca879

**Review by NipunaRanasinghe — APPROVED** [review]

**Comment by SasinduDilshara** [comment]
This PR will be wait until `data.csv` module migrate into java21.

**Review by RDPerera — DISMISSED** [review]
LGTM

**Comment by SasinduDilshara** [comment]
Updated the PR with lateset version

**Comment by SasinduDilshara** [comment]
> > Updated the PR with lateset version
> 
> There is a build failure in the PR
> 
> ```
> Bir missing for package :ballerina/data.csv
> Jar missing for package :ballerina/data.csv
> ```
> 
> I will check on this

Had a offline chat with team and decide to wait until newer minor version releases of the other modules

**Comment by SasinduDilshara** [comment]
> > > Updated the PR with lateset version
> > 
> > 
> > There is a build failure in the PR
> > ```
> > Bir missing for package :ballerina/data.csv
> > Jar missing for package :ballerina/data.csv
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > I will check on this
> 
> Had a offline chat with team and decide to wait until newer minor version releases of the other modules

close the PR for now and will reopen once all the modules are updated in the distribution

**Pull request closed without merging**